pilot-app/src/components/form/fields/wimChoiceRadio.vue

```diff
@@ -0,0 +1,61 @@
+﻿<template>
+    <div :class="classname">        
+        <label v-if="prefix(field)" v-html="prefix(field)"></label>
+        <span v-for="option in field.options" :key="fieldID(option)">
+            <input type="radio"
+                   v-on="eventHandler(field, 'handleChange')"
+                   v-model="field.value"
+                   class="radio"
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 13 spaces but found 19 spaces.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8170341)
```suggestion
                   v-bind:class="errorClass(field)"
```